packages/block-library/src/home-link/block.json

```diff
@@ -0,0 +1,39 @@
+{
+	"apiVersion": 2,
+	"name": "core/home-link",
+	"category": "design",
+	"parent": [ "core/navigation" ],
+	"title": "Home Link",
+	"keywords": [ "home", "menu", "navigation" ],
+	"textdomain": "default",
+	"attributes": {
+		"label": {
+			"type": "string"
+		},
+		"opensInNewTab": {
+			"type": "boolean",
+			"default": false
+		},
+		"rel": {
+			"type": "string"
+		},
+		"title": {
```

Comment:
Do we need this attribute? Could we make it be the title of the site by default and not editable?

Reply:
I asked for some quick feedback in https://wordpress.slack.com/archives/C02QB2JS7/p1620233144312500 and folks noted that it'd usually be "Home" instead of the site title.

Reply:
Hmm from https://make.wordpress.org/accessibility/handbook/content/link-destinations/#avoid-the-title-attribute-on-links maybe we can leave this unset for now?